AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/TaskTests/AWSAuthSignInPluginTests.swift

```diff
@@ -585,6 +585,56 @@ class AWSAuthSignInPluginTests: BasePluginTest {
         }
     }
 
+    /// Test a signIn with customAuthWIthoutSRP
+    ///
+    /// - Given: Given an auth plugin with mocked service. Returning a new challenge after confirm sign in is called
```

Comment:
nit: unnecessary extra "Given"
```suggestion
    /// - Given: An auth plugin with mocked service. Returning a new challenge after confirm sign in is called
```